yarn-project/pxe/src/check_oracle_version.ts

```diff
@@ -0,0 +1,36 @@
+import { keccak256String } from '@aztec/foundation/crypto';
```

Comment:
This should go in the `bin` directory.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/16435/commits/55dbabcae0a8ad44c7d878a61bee7d52dc06fe7d